client/app/dbaas/logs/detail/streams/alerts/home/alerts-home.html

```diff
@@ -8,7 +8,7 @@ <h6 class="oui-header_6">
     </h6>
     <p class="oui-paragraph mb-5" data-translate="streams_alerts_intro_text"></p>
 
-    <oui-action-menu text="{{ ::'create_alert' | translate}}" align="start">
+    <oui-action-menu text="{{ ::'create_alert' | translate}}" align="start" class="alerts">
```

Comment:
I noticed that you want the update the with of the `oui-dropdown__trigger` element containing inside the `<oui-action-menu>` component in order to be able to remove the truncated text right?

A proper solution is the replace the existing `<oui-action-menu>` with a [`<oui-select>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/select).

In that way you are able to get rid of the specific style that you want to apply.

Reply:
Thanks for your suggestion, i changed for <oui-select>.